projects/igniteui-angular/src/lib/grids/grid-base.directive.ts

```diff
@@ -327,7 +329,18 @@ export class IgxGridBaseDirective extends DisplayDensityBase implements
     }
 
     /**
-     *@hidden
+     * Emitted before filtering is performed through the UI.
```

Comment:
The event will be emitted no matter if it is UI or API change.

Reply:
Fixed.